qlib/workflow/recorder.py

```diff
@@ -227,18 +239,20 @@ def set_tags(self, **kwargs):
         else:
             mlflow.set_tags(dict(kwargs))
 
-    def delete_tag(self, key):
-        mlflow.delete_tag(key)
+    def delete_tags(self, *keys):

```

Comment:
![image](https://user-images.githubusercontent.com/465606/99142763-2358c400-2693-11eb-97c7-c272d028ab1a.png)

`len(keys) == 0` ? 
Why can't we use  mlflow.log_params(dict(kwargs)) direclty?